libbeat/outputs/elasticsearch/estest/estest.go

```diff
@@ -0,0 +1,23 @@
+package estest
```

Comment:
We could also have called `testing` as we do for other such packages.

Reply:
no big fan of 'testing' package, as `estest` some a little more vocal about helpers being about elasticsearch related tests. Fortunately we don't have many tests importing multiple testing packages ;)